src/hooks/useMaciPoll.ts

```diff
@@ -0,0 +1,53 @@
+import { getPoll, type Signer, type IGetPollData } from "maci-cli/sdk";
+import { useEffect, useMemo, useState } from "react";
+
+import { config } from "~/config";
+import { useEthersSigner } from "./useEthersSigner";
+
+export interface IUseMaciPollData {
+  isLoading: boolean;
```

Comment:
I think ok for demo purposes but I suspect ideally we'd have an `isError` state as well? Or is that case solved somewhere else?

Reply:
I don't think we need to cover all the cases for demo. The only one thing can happens if user have issues with internet.